macros/utils/table_operations/create_or_replace.sql

```diff
@@ -25,3 +25,9 @@
     {% do elementary.run_query(dbt.create_table_as(temporary, relation, sql_query)) %}
     {% do adapter.commit() %}
 {% endmacro %}
+
+{% macro athena__create_or_replace(temporary, relation, sql_query) %}
+    {% do dbt.drop_relation_if_exists(relation) %}
+    {% do elementary.run_query(dbt.create_table_as(temporary, relation, sql_query)) %}
+    {% do adapter.commit() %}
```

Comment:
commit is not strictly needed in athena, as the concept of commit doesn't exist. Is this really necessary?
I mean this macro is really athena specific.

Reply:
sure, we can remove it. Maybe someday Athena will have commits and it will just work :)).

Reply:
It's going to be a bit day, as we have a full transactional datalakehouse :)

Reply:
fixed